### DISCUSSION TITLE
Stellarium script "Star distance filter"

### DISCUSSION THREAD

**Discussion by foxjony** [Stellarium/stellarium#2690] [General]
Stellarium allows you to use scripts.
Can someone answer a few questions?

1. How to connect and use the script?
2. How to create your own script?
3. From what base are the coordinates of the stars taken?
4. Is there a script to filter stars by distance from the Sun? (for example, show only nearby stars in the sky with a distance of no more than 100 light years from the Sun)
5. Is there such a script for Celestia?

**Comment by gzotti** [comment]
1-3: See User Guide
4: No
5: Ask Celestia
